main.go

```diff
@@ -20,7 +20,7 @@ var handshakeConfig = goPlugin.HandshakeConfig{
 
 func main() {
 	logCtx := log.WithFields(log.Fields{"plugin": "trafficrouter"})
-	log.SetLevel(log.DebugLevel)
+	log.SetLevel(log.InfoLevel)
```

Comment:
I noticed some other plugins left this at Debug... but I don't think we should be emitting our Debug logs.